enterprise/frontend/src/metabase-enterprise/moderation/constants.js

```diff
@@ -1,7 +1,20 @@
+export const MODERATION_STATUS = {
+  verified: "verified",
+};
+
 export const ACTIONS = {
   verified: {
     type: "verified",
-    icon: "verified",
-    color: "brand",
+    icon: {
+      name: "verified",
+      color: "brand",
+    },
+  },
+  null: {
+    type: "null",
+    icon: {
+      name: "close",
+      color: "text-light",
+    },
```

Comment:
What does "null" mean here? Should we maybe add a comment or maybe use a constant with a verbose name? I mean something like:

```js
// let's say "null" means "remove"
const REMOVE_ACTION_TYPE = "null"

export const ACTIONS = {
   [REMOVE_ACTION_TYPE]: {
       type: REMOVE_ACTION_TYPE,
       // other things
   }
}
```

Reply:
good call out -- this `ACTIONS` object is kind of confusing since it really doesn't do much. I renamed it to be explicitly for icons and then I tried to make the biz logic of a "removed review" status clearer. Also, config objects passed to functions are kind of annoying to me, so I rewrote it to force you to explicitly ask for a "removed review" icon explicitly, since it shouldn't be used much (or at all) besides for revision events in the timeline. 

How's that?